tracer/test/test-applications/integrations/Samples.Selenium/ci.runsettings

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+	<DataCollectionRunSettings>
+		<DataCollectors>
+			<DataCollector friendlyName="Code Coverage">
+				<Configuration>
+					<Format>xml</Format>
+					<CodeCoverage>
+						<!-- Switching to static instrumentation (dynamic instrumentation collides with dd-trace instrumentation) -->
```

Comment:
nit: might be nice to note _why_ we have to do this (i.e. because otherwise it doesn't work) 😅

Reply:
done in [a93292f](https://github.com/DataDog/dd-trace-dotnet/pull/5633/commits/a93292f4bebf0023b339f7a57546965a9edbb35d)